GoogleForCreators/web-stories-wp
# Canvas: Update zoom dropdown to align with help center

## Context

Using the design system in the editor. No styles in the carousel use the deprecated theme.

![image](https://user-images.githubusercontent.com/22185279/112899560-87f34900-909f-11eb-9fc2-215e127b95e2.png)
[Designs](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=274%3A250)

Chat with designers about the chevron: https://xwp.slack.com/archives/C01ESBB8ZN1/p1617050704003700

Tasks:
- [x] Update the dropdown for zoom to use the new design system dropdown
- [x] Make sure the design corresponds to the Figma guide
- [x] Make sure the zoom dropdown and help center toggle are perfectly aligned as in Figma
- [x] Make sure that there are no references to `DEPRECATED_THEME`

## Summary

Updated things:
- Zoom dropdown uses the `DropDown` from the design system

## Relevant Technical Choices

- Added a `showChevron` boolean to give us the ability to hide the chevron like in the designs

## To-do

- [x] talk with design about whether to keep checkmark in the menu or not
- [ ] fix checkmark icon for dropdown in RTL (not related to this pr)

## User-facing changes

|Before|After|After RTL|
|--|--|--|
|![image](https://user-images.githubusercontent.com/22185279/112899669-af4a1600-909f-11eb-88a8-6a0c0e313e8e.png)|![image](https://user-images.githubusercontent.com/22185279/112902224-10271d80-90a3-11eb-9b60-bb90b9c41502.png)|![image](https://user-images.githubusercontent.com/22185279/112902264-1ae1b280-90a3-11eb-8a9f-46ae2d64a8b1.png)|

## Testing Instructions

1. Make sure `Canvas Zoom` experiment is turned on
2. Go to editor
3. See the zoom dropdown in the lower right hand corner of the canvas
4. Play with it - should be keyboard accessible too

### QA






This PR can be tested by following these steps:

1. Make sure `Canvas Zoom` experiment is turned on
2. Go to editor
3. See the zoom dropdown in the lower right hand corner of the canvas
4. Play with it - should be keyboard accessible too

### UAT






## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

no

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #6912
